.github/workflows/ci.yaml

```diff
@@ -52,10 +51,53 @@ jobs:
       - name: Install required Python packages
         run: |
           python -m pip install --upgrade pip
-          python -m pip install tox build
-
-      - name: Build the Python package
-        run: python -m build
+          python -m pip install tox
 
       - name: Run tox
         run: tox -r
+
+  build-dist:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Fetch sources
+        uses: actions/checkout@v3
+        with:
+          submodules: true
+
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: "3.10"
```

Comment:
I'm not a 100% sure but the wheel doesn't carry information about the python version (only if it's python 2 or 3, for example: `frequenz_api_microgrid-0.11.0a1-py3-none-any.whl`). Since we don't have different code for different python versions, I don't see how making different packages could make any sense, we just need to make sure our **single code base** stays compatible with all Python versions we want to support.

I will try to verify this though.

Reply:
Okay. I think it's okay if we get this in, and fix things in a future PR, if required at all.

Reply:
Confirmed: we **can** ship different wheels for different python versions (if we use, for example `-py37`) but we **don't have to** (because as I said we support all the different Python versions through the same code base). The meaning of `-py3` is *any Python 3 version* (then the metadata inside the wheel can narrow this down).

Some useful links for context:
* Wheel specs: https://packaging.python.org/en/latest/specifications/binary-distribution-format/
* Wheel PEP: https://peps.python.org/pep-0427/
* Compatibility Tags for Built Distributions: https://peps.python.org/pep-0425/#python-tag